Tasks/AzureWebAppV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -18,6 +18,8 @@
   "loc.input.help.slotName": "Enter or Select an existing Slot other than the Production slot.",
   "loc.input.label.package": "Package or folder",
   "loc.input.help.package": "File path to the package or a folder containing app service contents generated by MSBuild or a compressed zip or war file.<br />Variables ( [Build](https://docs.microsoft.com/vsts/pipelines/build/variables) | [Release](https://docs.microsoft.com/vsts/pipelines/release/variables#default-variables)), wildcards are supported. <br/> For example, $(System.DefaultWorkingDirectory)/\\*\\*/\\*.zip or $(System.DefaultWorkingDirectory)/\\*\\*/\\*.war.",
+  "loc.input.label.customWarName": "Custom war name",
+  "loc.input.help.customWarName": "Specify custom folder name you want to deploy to. <br/>If the field is empty, package is deployed to <appname>.azurewebsites.net/<warpackagename><br/> If ROOT is entered, package is deployed to <appname>.azurewebsited.net <br/>else it is deployed to <appname>.azurewebsited.net/<customWarName>",
```

Comment:
Can you please get the sign off for resources from @N-Usha?

Reply:
done